src/app/components/onboarding/login/login.component.ts

```diff
@@ -19,12 +21,18 @@ export class LoginComponent implements OnInit, OnDestroy {
   loginRequest = false;
   private subs = new SubSink();
   hide = true;
+  language: string;
+  captchaSuccess = false;
+  captchaKey = environment.captcha.key;
+  script: any;
 
   constructor(formBuilder: FormBuilder,
```

Comment:
Das Captcha sollte glaub ich ins Registrierungsformular und nicht ins Login :)

Reply:
Hatte mit Michael besprochen dass ich es hier rein mach. Geht aber natürlich auch auf der anderen Seite

Reply:
brauchen wir das im Login @MichaelSpielmann ? Hätte es jetzt auch eher auf der Registerseite verortet um Spam-Registrierungen zu vermeiden.. beim login is ja wurscht denk ich, oder?